app/components/partner/client-info-plan/styles.scss

```diff
@@ -0,0 +1,41 @@
+.plan {
+  display: flex;
+}
+
+.plan-type {
+  border-radius: 0.2rem;
+  padding: 0.2rem 0.6rem;
+  color: $white;
+  text-transform: capitalize;
+  line-height: 1.2;
+  font-weight: bold;
+  border: 1px solid transparent;
+
+  &.per-app {
+    color: #5995ef;
+    background-color: #F2F7FF;
+    border-color: darken(#F2F7FF, 1%);
+  }
+
+  &.per-scan {
+    color: #57bf72;
+    background-color: #f0f9f2;
+    border-color: darken(#f0f9f2, 1%);
+  }
+}
```

Comment:
There are few styling changes, can you copy from here: https://github.com/appknox/irene/blob/a714cbe85b/app/components/partner/client-info/styles.scss#L82-L102

Reply:
sure 👍

Reply:
@guganabu can you verify this? styles are not updated yet